imagersite/imager_images/tests.py

```diff
@@ -110,3 +116,336 @@ def test_many_to_many(self):
         self.album2.photos.add(*self.photos)
         for p1, p2 in zip(self.album1.photos.all(), self.album2.photos.all()):
             self.assertEqual(p1, p2)
+
+
+class LibraryPage(TestCase):
```

Comment:
It is traditional to name test cases by including the word `Test` or `TestCase` in the name:

```python
class LibraryPageTestCase(TestCase):
    #...
```